pkg/cmd/cluster/cluster_attach.go

```diff
@@ -55,7 +56,12 @@ func newCmdClusterAttach(ctx api.Context) *cobra.Command {
 			case 0:
 				// No matching linked cluster, one matching cluster.
 				if matchingConf != nil {
-					return manager.Attach(matchingConf)
+					err = manager.Attach(matchingConf)
+					if err != nil {
+						return err
+					}
+					ctx.Logger().Infof("You are now attached to cluster %s", args[0])
```

Comment:
`args[0]` might only be a short ID prefix, we'd better display the full one here : `config.NewCluster(matchingConf).ID()`